library/Icinga/Authentication/Auth.php

```diff
@@ -378,7 +378,6 @@ public function setupUser(User $user)
 
         if ($config->get('global', 'config_backend', 'db') !== 'none') {
```

Comment:
That's obsolete now, remove it.

Reply:
The option `config_backend` is also referenced multiple times in various other places.